### ISSUE TITLE
Mentelmäosaan linkki Vuorijärveen?

### ISSUE THREAD

**Issue by Kirde80** [hrmJ/100-tapaa-indikoida-topiikkia#25]
Tällainen oli muistiinpanoissa. Täytyy muistaa tsekata, liitämmekö maininnan artikkeliin. 

Käyttämämme menetelmä muistuttaa prosessina Vuorijärven analyysiprosessin vaiheita: sivut (116-117).

**Comment by hrmJ** [comment]
Joo,  täytyy se jotenkin.. Mä teen hei sen kuvion sinne metodiin ja samalla laitan maininnan tosta, jahka oon ensin vähän silmäillyt
